modules/develop/pages/produce-data/configure-producers.adoc

```diff
@@ -213,17 +217,26 @@ in one place, while also enabling features like schema evolution.
 [[broker-timestamps]]
 == Broker timestamps
 
-Redpanda employs a unique strategy to help ensure the accuracy of retention operations. In this strategy, closed segments are only eligible for deletion when the age of all messages in the segment exceeds a xref:manage:cluster-maintenance/disk-utilization.adoc#set-time-based-retention[configured threshold]. However, when a producer sends a message to a topic, the timestamp set by the producer may not accurately reflect the time the message reaches the broker. To address this time skew, each time a producer sends a message to a topic, Redpanda records the broker's system date and time in the `broker_timestamp` property of the message. This property helps maintain accurate retention policies, even when the message's creation timestamp deviates from the broker's time.
+Redpanda employs a unique strategy to help ensure the accuracy of retention operations. In this strategy, closed segments are only eligible for deletion when the age of all messages in the segment exceeds a configured threshold. However, when a producer sends a message to a topic, the timestamp set by the producer may not accurately reflect the time the message reaches the broker. To address this time skew, each time a producer sends a message to a topic, Redpanda records the broker's system date and time in the `broker_timestamp` property of the message. This property helps maintain accurate retention policies, even when the message's creation timestamp deviates from the broker's time.
+
+ifndef::env-cloud[]
+See also:
+
+- xref:manage:cluster-maintenance/disk-utilization.adoc#set-time-based-retention[Set time-based retention]
 
+ifndef::env-cloud[]
 === Configure broker timestamp alerting
 
 Each time a broker receives a message with a skewed timestamp that is outside a configured range, Redpanda increments the xref:reference:internal-metrics-reference.adoc#vectorized_kafka_rpc_produce_bad_create_time[`vectorized_kafka_rpc_produce_bad_create_time`] metric. Two cluster properties control this range. The minimum accepted value for both of these properties is five minutes. Any attempt to set a value lower than that is rejected by Redpanda. 
 
 * `log_message_timestamp_alert_before_ms`: Defines the allowed skew before the broker's time. This check is effectively disabled when the value is set to `null`. Minimum: `300000 ms` (5 minutes), Default: `null`.
 * `log_message_timestamp_alert_after_ms`: Defines the allowed skew after the broker's time. There is no way to disable this check. Minimum: `300000 ms` (5 minutes), Default: `7200000 ms` (2 hours).
 
+endif::[]
+
 === Disable broker timestamp retention
 
+// For cloud there must be restrictions on Admin API usage. We should find out and omit any references to unsupported endpoints.
```

Comment:
@micheleRP we probably need a separate issue for this. Do you know if there are restrictions/what they are?

Reply:
@JakeSCahill and @micheleRP,  I've added a condition to exclude this subsection from the Cloud docs. Is there a separate pull request for other mentions of the Admin API in single-sourced content?